tests/installation/ipxe_install.pm

```diff
@@ -141,6 +171,21 @@ END_BOOTSCRIPT
     diag "$response->{status} $response->{reason}\n";
 }
 
+sub enter_o3_ipxe_boot_entry {
+    ipmitool('chassis power reset') unless check_screen([qw(o3-ipxe-menu ipxe-boot-failure)], 180);
+    assert_screen('o3-ipxe-menu', 300);
```

Comment:
With default TIME_SCALE 3, it wil give 900 seconds(15 minutes) to asset ipxe menu. Is it really needed? How about fail earlier?

Reply:
it depends on the restart time including power off and power on. I cannot get an acurate time from a test case, https://openqa.opensuse.org/tests/3446335#step/ipxe_install/22, but the time seems to be shorter than 900s. Let's reduce to 210 and keep eyes on it in future tests.